modules/sno-du-prerequisites.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-prerequisites_{context}"]
+= Prerequisites
+
+* A namespace for the cluster must exist for the unique
```

Comment:
To be more specific this namespace must exist on the hub cluster from which installation is being performed.

Reply:
changed